### ISSUE TITLE
Frida not detected

### ISSUE THREAD

**Issue by Bilecen** [scottyab/rootbeer#206]
Hi, there is a frida vulnerability on the android side, can you help with this issue?

**Comment by skbrijesh13** [comment]
Right, we are also facing this vulnerability, do we have any update if this is being worked upon?
